### PR TITLE
navbar polish

### PR DESCRIPTION
**Pull request by ajb**

### CODE REVIEW

**Comment by jrubenoff** [comment]
Fixed navbars really suck in mobile browsers. Maybe we could only add fixed positioning for wide viewports?

File: vendor/assets/stylesheets/dvl/components/dropdowns.scss [hunk 1]
```diff
@@ -117,6 +117,14 @@
   margin-bottom: 0.25rem;
 }
 
+// Modifier for dropdown header, used in navbar
+.dropdown_header_strong {
```
Comment (jrubenoff) [inline vendor/assets/stylesheets/dvl/components/dropdowns.scss]: Seems like it could be more semantic... `.dropdown_header_nav`?

Comment (ajb) [inline vendor/assets/stylesheets/dvl/components/dropdowns.scss]: Was struggling with naming, was going to ask. Sounds good.

**Comment by ajb** [comment]
> Fixed navbars really suck in mobile browsers. Maybe we could only add fixed positioning for wide viewports?

I tend to disagree... fixed navbars make it feel more like a "mobile app", and I particularly like the effect when the mobile browser hides the URL bar:



But I can defer to you (and bootstrap, apparently) on this one.

**Pull request merged**